src/main/java/net/sourceforge/argparse4j/impl/type/EnumStringArgumentType.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2011 Tatsuhiro Tsujikawa
```

Comment:
Since this file is solely written by you, you should put your name and copyright year 2015.